src/lib/react-server-route.js

```diff
@@ -6,11 +6,53 @@ import { match, RouterContext } from 'react-router';
+var routes = require('../routes.js');
 
 module.exports = function(req, res, next) {
   var location = req.url;
 
+  var metaTitle = "Mozilla’s Policy & Advocacy Program - Home";
+  var metaSiteName = "Mozilla Advocacy";
+  var metaUrl = "http://advocacy.mozilla.org/";
+  var metaDesc = "We are building a global movement to protect the free and open Web.";
+  var metaImage = "common/assets/logo-og.png";
+  var twitterImage = "";
+  var desc = "We Are Building a Global Movement to Protect the Free and Open Web Mozilla supports the heroes of the Web — the developers, advocates and ...";
+  var title = "Mozilla Advocacy";
+
+  if (location.indexOf('/encrypt/') !== -1) {
+    metaTitle = "Encryption matters";
+    metaSiteName = "Mozilla Advocacy - Encryption";
+    metaUrl = "https://advocacy.mozilla.org/encrypt/";
+    metaDesc = "Encryption needs you! Sign Mozilla's pledge to stand up for encryption and a more secure Web at advocacy.mozilla.org/encrypt";
+    metaImage = "https://advocacy.mozilla.org/assets/img/share/Encryption-facebook-b5-06.png";
+    twitterImage = "https://advocacy.mozilla.org/assets/img/share/Encryption-twitter-b5-06.png";
+    desc = "Your online privacy depends on encryption. Learn more about how encryption works, why it's essential to a strong Web, and why it's worth protecting.";
+    title = "Learn about encryption - Mozilla Advocacy";
+  }
+
+  if (location.indexOf('/encrypt-hard-drive/') !== -1) {
+    metaTitle = "Encrypt Your Hard Drive";
+    metaSiteName = "Learn about encryption - Mozilla Advocacy";
+    metaUrl = "https://advocacy.mozilla.org/encrypt-hard-drive/";
+    metaDesc = "Keep all that personal data safer: Take a few minutes encrypt your hard drive. The web will thank you. @Mozilla has a step-by-step guide to help.";
+    metaImage = "https://advocacy.mozilla.org/assets/filevault_illustration.jpg";
+    twitterImage = "https://advocacy.mozilla.org/assets/filevault_illustration.jpg";
+    desc = "We Are Building a Global Movement to Protect the Free and Open Web Mozilla supports the heroes of the Web — the developers, advocates and ...";
+    title = "Encrypt Your Hard Drive";
+  }
+
+  if (location.indexOf('/stay-secure/') !== -1) {
+    metaTitle = "Update Your Software";
+    metaSiteName = "Learn about encryption - Mozilla Advocacy";
+    metaUrl = "https://advocacy.mozilla.org/stay-secure/";
+    metaDesc = "Keep all that personal data safer: Take a few minutes and update the software on all your devices -- including phones, tablets, and computers. The web will thank you. Update today with @Mozilla’s help.";
+    metaImage = "https://advocacy.mozilla.org/assets/01_SoftwareUpdateImage-Facebook.png";
+    twitterImage = "https://advocacy.mozilla.org/assets/02_SoftwareUpdateImage-Twitter.png";
+    desc = "We Are Building a Global Movement to Protect the Free and Open Web Mozilla supports the heroes of the Web — the developers, advocates and ...";
+    title = "Update Your Software";
+  }
```

Comment:
All three of these if statements are for specific campaigns. They match their older copy and pasted index files.